1_fetch/src/sbtools_utils.R

```diff
@@ -0,0 +1,113 @@
+#' Login to ScienceBase using the "cidamanager" user credentials from dssecrets package
+#'
+#' @details To use `dssecrets`, ensure you have file system access to the cidamanager
+#' secret among the files where `dssecrets` is installed.
+#' If you prefer, you can bypass `dssecrets` by manually calling
+#' sbtools::authenticate_sb('[my_sb_user]'),
+#' provided your account has permission to access the SB item(s).
+sb_secret_login <- function(){
+  if (!sbtools::is_logged_in()){
+    sb_secret <- dssecrets::get_dssecret("cidamanager-sb-srvc-acct")
+    sbtools::authenticate_sb(username = sb_secret$username, password = sb_secret$password)
+  }
+}
+
+#' @return a tibble with columns filename (as on SB), date_uploaded (as on SB),
+#'   date_checked (time of query to SB), and fetch_datestamp (date_uploaded of
+#'   the file on SB at the time we fetched it into this repo)
```

Comment:
I think these function descriptions should be bumped down a few lines -- seems like they go with `update_sb_status()` rather than `read_sb_status()`

Reply:
I see how you might think that. I've beefed up the documentation for this function and the next one.